src/api/time/IndependentTimeContext.js

```diff
@@ -226,10 +226,12 @@ class IndependentTimeContext extends TimeContext {
     const previousClock = this.activeClock;
     if (previousClock) {
       previousClock.off('tick', this.tick);
+      previousClock.stop();
```

Comment:
Maybe we _don't_ want to do this. You might have multiple time conductors sharing a clock instance and you wouldn't want to stop it in that case. The clocks should stop themselves when they have no more listeners

Reply:
Yeah I see what you're saying